File: HackerRank/Python/Easy/E0082.py
# Problem Statement: https://www.hackerrank.com/challenges/np-inner-and-outer/problem

import numpy as np

A = np.array(input().split(), dtype='int32')
B = np.array(input().split(), dtype='int32')
print(np.inner(A, B), np.outer(A, B), sep='\n')

File: HackerRank/Algorithms/Easy/E0082.cpp
/*
Problem Statement: https://www.hackerrank.com/challenges/lonely-integer/problem
*/

#include <iostream>
#include <vector>

using namespace std;

int lonelyinteger(vector<int> a) {
	int unique = 0;
	for (int i = 0; i < a.size(); i++)
		unique ^= a[i];
	return unique;
}

int main() {
	int n;
	cin >> n;
	vector<int> a(n);
	for (int i = 0; i < n; i++)
		cin >> a[i];
	cout << lonelyinteger(a);
}